### ISSUE TITLE
Load mask might not appear at first time in some scenario

### ISSUE THREAD

**Issue by DaniilVeriga** [extnet/Ext.NET#852]
http://forums.ext.net/showthread.php?59826

Probably, same or similar issue:
https://www.sencha.com/forum/showthread.php?299670

**Update:** Sencha issue above is still open after 6.0.1 release.

**Comment by fabriciomurta** [comment]
**Update:** Issue still open after ExtJS 6.0.2 release.

**Comment by fabriciomurta** [comment]
Working test case:

```
@{
    Layout = null;
}
<!DOCTYPE html>
<html>
<head>
    <meta name="viewport" content="width=device-width" />
    <title></title>
</head>
<body>
    @Html.X().ResourceManager().Theme(Theme.Gray)

    @(
    Html.X().Window().Width(400).Height(400).Layout(LayoutType.Fit).Items(
        Html.X().GridPanel()
            .ID("ServiceGroupId")
            .Store(
                Html.X().Store()
                    .Model(
                        Html.X().Model()
                            .IDProperty("CKEY")
                            .Fields(
                                Html.X().ModelField().Name("CKEY"),
                                Html.X().ModelField().Name("CVALUE")
                            )
                    )
                    .PageSize(10)
                    .Proxy(Html.X().AjaxProxy()
                        .Url(Url.Action("c59826_comboValues"))
                        .Reader(Html.X().JsonReader().RootProperty("data"))
                    )
                    .Parameters(p => p.Add(new StoreParameter("ctp", "T8D", ParameterMode.Auto)))
            )
            .ColumnModel(
                Html.X().Column().Text("CKEY").DataIndex("CKEY"),
                Html.X().Column().Text("CVALUE").DataIndex("CVALUE")
            )
        )
)
</body>
</html>
```

Controller:
```csharp
public ActionResult c59826_Index() { return View(); }
public ActionResult c59826_comboValues(StoreRequestParameters requestParameters)
{
    var list = new List<object>();
    list.Add(new { CKEY = "CKEY1", CVALUE = "cv1" });
    list.Add(new { CKEY = "CKEY2", CVALUE = "cv2" });
    list.Add(new { CKEY = "CKEY3", CVALUE = "cv3" });
            
    System.Threading.Thread.Sleep(2000);

    return this.Store(list);
}
```

**Comment by fabriciomurta** [comment]
The corresponding Sencha issue is still open so it is probably not the same problem.

**Comment by fabriciomurta** [comment]
**Update:** Sencha issue allegedly fixed by the time ExtJS 6.2.1 was released.

**Comment by fabriciomurta** [comment]
**Update:** Sencha issue _EXTJS-17254_ allegedly fixed in ExtJS 7.0.0 release.
